Platforms/QemuQ35Pkg/CfgData/QemuQ35PkgCfgData.xml

```diff
@@ -0,0 +1,12 @@
+<ConfigSchema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"

```

Comment:
Maybe I missed it, but should we replace the existing yaml files?

Reply:
Yes, there is some cleanup here to not use the old yaml system that I need to add.